src/xunit.performance.api/ScenarioConfiguration.cs

```diff
@@ -10,17 +12,22 @@ public sealed class ScenarioConfiguration
         /// <summary>
         /// Initializes a new instance of the ScenarioConfiguration class.
         /// </summary>
-        /// <param name="timeSpam">The amount of time to wait for one iteration process to exit.</param>
+        /// <param name="timeSpan">The amount of time to wait for one iteration process to exit.</param>
         /// <param name="iterations">Number of times a benchmark scenario process will be executed.</param>
-        public ScenarioConfiguration(TimeSpan timeSpam, int iterations = 10)
+        /// <param name="validExitCodes">
+        /// A collection of exit codes that indicates success when the benchmark scenario process terminates.
+        /// If this parameter is not specified, then the only valid exit code will 0.
+        /// </param>
+        public ScenarioConfiguration(TimeSpan timeSpan, int iterations = 10, IEnumerable<int> validExitCodes = null)
```

Comment:
I think you should separate this out into a mutable and immutable version.  That way, you can have mutable state before the harness method is invoked, and you don't need to make changes to the constructor every time you want to have a new parameter.